data/players.yaml

```diff
@@ -4932,3 +4932,16 @@
   twitch:
     - https://www.twitch.tv/twestaoe
   id: 403
+- name: PhilippHJS
+  country: de
+  aoeelo: 908
+  steam:
+    - 76561198014542759
```

Comment:
Don't think that's necessary

Reply:
Just copy pasted, can remove, no problem.